cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -68,6 +69,7 @@ parameter_types! {
 	pub const MaxAssetsIntoHolding: u32 = 64;
 	pub TreasuryAccount: AccountId = TREASURY_PALLET_ID.into_account_truncating();
 	pub RelayTreasuryLocation: MultiLocation = (Parent, PalletInstance(rococo_runtime_constants::TREASURY_PALLET_ID)).into();
+	pub SiblingPeople: MultiLocation = (Parent, Parachain(1004)).into();
```

Comment:
I think instead of `1004` we should use rococo's system parachain constant, I would add it like here:
https://github.com/paritytech/polkadot-sdk/pull/2281/files#diff-723e030bc6a22c68b3606b0463863382f2b3c82af20c7b36da8d889199e1b86eR115-R116

Reply:
Yep, let's wait until https://github.com/paritytech/polkadot-sdk/pull/2281 is merged - I'll do final polishing afterwards

Reply:
Actually, let's add it here - I've forgot that this PR is based on master, so IIUC there's nothing wrong with adding it here